boards/arm/power_module/power_module.dts

```diff
@@ -79,6 +79,7 @@
     apb2-prescaler = <1>;
 };
 
+/*
```

Comment:
is this adc fake? or was this the thing you needed to do to make CI build. dyk what the root issue on that was?

Reply:
Nah its real af and has never caused problems up until now. We don't really use it anyways since the INA219s also get the job done, which is why Im comfortable commenting it out for now and uncommenting it later